src/app/components/cookie-policy/cookie-policy.component.html

```diff
@@ -16,150 +17,154 @@ <h1 class="govuk-heading-xl" id="cookies">Cookies</h1>
-        <p>We use authentication cookies to identify you when you return to the service.</p>
-        <table class="govuk-table">
-          <thead class="govuk-table__head">
+            </thead>
+            <tbody>
+            <tr class="govuk-table__row" *ngFor="let cookie of cookiesByCat(SESSION)">
+              <td class="govuk-table__cell">{{cookie.name}}</td>
+              <td class="govuk-table__cell">{{cookie.purpose}}</td>
+              <td class="govuk-table__cell">{{cookie.expires}}</td>
+            </tr>
+            </tbody>
+          </table>
+        </ng-container>
+        <ng-container *ngIf="countCookies(IDENTIFY)">
+          <h3 class="govuk-heading-m" id="4-to-identify-you-when-you-come-back-to-the-service">To identify you when
+            you come back to the service</h3>
+          <p>We use authentication cookies to identify you when you return to the service.</p>
+          <table class="govuk-table">
+            <thead class="govuk-table__head">
             <tr class="govuk-table__row">
               <th class="govuk-table__header">Name</th>
               <th class="govuk-table__header">Purpose</th>
               <th class="govuk-table__header">Expires</th>
             </tr>
-          </thead>
-          <tbody>
-            <tr class="govuk-table__row">
-              <td class="govuk-table__cell">__auth-token</td>
-              <td class="govuk-table__cell">Identifies you to the service</td>
-              <td class="govuk-table__cell">When you close your browser</td>
-            </tr>
-          </tbody>
-        </table>
-        <h3 class="govuk-heading-m" id="5-to-make-the-service-more-secure">5. To make the service more secure</h3>
-        <p>We set cookies which prevent attackers from modifying the contents of the other cookies we set. This makes the service more secure and protects your personal information.</p>
-        <table class="govuk-table">
-          <thead class="govuk-table__head">
+            </thead>
+            <tbody>
+            <tr class="govuk-table__row" *ngFor="let cookie of cookiesByCat(IDENTIFY)">
+              <td class="govuk-table__cell">{{cookie.name}}</td>
+              <td class="govuk-table__cell">{{cookie.purpose}}</td>
+              <td class="govuk-table__cell">{{cookie.expires}}</td>
+            </tr>
+            </tbody>
+          </table>
+        </ng-container>
+        <ng-container *ngIf="SECURITY">
+          <h3 class="govuk-heading-m" id="5-to-make-the-service-more-secure">To make the service more secure</h3>
+          <p>We set cookies which prevent attackers from modifying the contents of our pages and which prevent
+            unauthorised access to the service. This
+            makes the service more secure and protects your personal information.</p>
+          <table class="govuk-table">
+            <thead class="govuk-table__head">
             <tr class="govuk-table__row">
```

Comment:
Sixth repetition of this bit...

Reply:
It's not worth the effort of making a component for this. HTML is a perfectly reasonable Language for Marking up HyperText, and I'm not translating HTML into typescript for no reason

Reply:
_This_ is already an Angular component utilising bindings. It's just that this Angular component has 6 identical tables. 🤷‍♂️